### PR TITLE
ENT-3228 : Candlepin project restructuring

### PR DESCRIPTION
**Pull request by wolfdale**
### *********************DO NOT MERGE*********************

Changes
- Updated the candlepin deployment path.

### CODE REVIEW

File: tasks/user.yml
Comment (nikosmoum) [inline tasks/user.yml:53]: @wolfdale This will make it so that `vagrant up` will only work with master branch. Can we make this conditional on whether the deployment dir exists, and if not, running with the old path?

**Review by nikosmoum — CHANGES_REQUESTED** [review]

File: tasks/user.yml
Comment (wolfdale) [reply tasks/user.yml:53]: I don't think yaml file support conditional statements. I could find other alternatives.

My assumption was that we perform ```vagrant up``` on master branch only & then switch branches that we want to work with. Since no matter which branch we are, only latest ansible roles will get picked. This is the reason were we want to have multiple vagrant files & start tagging ansible-role repo with each release.

Comment (nikosmoum) [reply tasks/user.yml:53]: you could make a conditional statement within the shell command perhaps, but you are right. we only support master for now.

Let's make that clear by adding a note in the README.md of this project. (that this ansible role only works with master candlepin, and you would have to manually switch branch and re-configure/run candlepin as needed for older branches)

**Review by nikosmoum — APPROVED** [review]
ACK. will merge after the main PR is merged

**Pull request merged**
